### PR TITLE
Fix warning implementation_imports linter warning

### PR DESCRIPTION
**Pull request by T-moz**
Lint: https://dart-lang.github.io/linter/lints/implementation_imports.html

### CODE REVIEW

File: example/lib/generated_plugin_registrant.dart [hunk 1]
```diff
@@ -5,8 +5,7 @@
 // ignore_for_file: directives_ordering
 // ignore_for_file: lines_longer_than_80_chars
 
-// ignore: implementation_imports
-import 'package:flutter_segment/src/segment_web.dart';
```
Comment (danielgomezrico) [inline example/lib/generated_plugin_registrant.dart]: Why adding a file that exports the same import will fix the warning? is it the real solution for this?

Comment (danielgomezrico) [reply example/lib/generated_plugin_registrant.dart]: @T-moz Ping

**Review by cdmunoz — CHANGES_REQUESTED** [review]
This changes don't have any relationship with fixing the linter.
In addition, the code base is behind master and other changes that need to be updated to do a proper review.

**Review by cdmunoz — APPROVED** [review]
LGTM 👌🏻

**Comment by danielgomezrico** [comment]
Fix https://github.com/la-haus/flutter-segment/issues/24

**Pull request merged**